orchestrator/graphql/autoregistration.py

```diff
@@ -44,6 +43,15 @@ def is_enum(field: Any) -> bool:
     return inspect.isclass(field) and issubclass(field, Enum)
 
 
+def create_strawberry_enums(model: Type[DomainModel], strawberry_enums: EnumDict) -> EnumDict:
+    enums = {
+        key: field
+        for key, field in model._non_product_block_fields_.items()
+        if is_enum(field) and is_not_strawberry_enum(key, strawberry_enums)
+    }
+    return strawberry_enums | {key: create_strawberry_enum(field) for key, field in enums.items()}
```

Comment:
The field can be turned into an enum in the same comprehension, to eliminate the extra comprehension at the bottom

```suggestion
    enums = {
        key: create_strawberry_enum(field)
        for key, field in model._non_product_block_fields_.items()
        if is_enum(field) and is_not_strawberry_enum(key, strawberry_enums)
    }
    return strawberry_enums | enums
```